moped-editor/src/views/dashboard/DashboardStatusModal.js

```diff
@@ -58,8 +58,15 @@ const DashboardStatusModal = ({
         fullWidth
         maxWidth={"md"}
       >
-        <DialogTitle className={classes.dialogTitle}>
-          <h3>{`Status update - ${projectName}`}</h3>
+        <DialogTitle
+          sx={{
+            display: "flex",
+            justifyContent: "space-between",
+            alignItems: "center",
+          }}
```

Comment:
removing the custom styles means we need to add in this sx prop so that the close button will be aligned nicely